tests/api_connexion/endpoints/test_extra_link_endpoint.py

```diff
@@ -95,7 +94,7 @@ def _create_dag():
         with DAG(
             dag_id="TEST_DAG_ID",
             default_args=dict(
-                start_date=days_ago(2),
+                start_date=datetime.datetime(2022, 1, 1),
```

Comment:
I suspect we should move away from `datetime.datetime` to `pendulum.datetime` for `start_date` values in unit tests too based on the conversation in #21646. @eladkal WDYT?

Reply:
Do you mean just for tests or also for example dags?
I have no objection.

Reply:
For unit tests as well as example DAGs for consistency in tests, examples, docs, etc. The core example DAGs have been transitioned but the provider ones haven't yet.

Reply:
I too think pendulum is a better alternative for datetime

Reply:
Then yeah lets do that

Reply:
Of course! Not a problem at all

Reply:
ok thanks

Reply:
For tests specifically I actually prefer `datetime.datetime` since it better ensures we are coercing timezones correctly internally.

Reply:
That one is also done in the previous pull request. I guess my task is done so please let me know if there is anything more to update.